### PR TITLE
log all fragments received by the pool

### PR DESCRIPTION
**Pull request by eugene-babichenko**

### CODE REVIEW

**Review by dkijania — APPROVED** [review]

File: jormungandr/src/fragment/pool.rs [hunk 1]
```diff
@@ -81,7 +81,16 @@ impl Pools {
...
 
         tracing::debug!(origin = ?origin, "received {} fragments", fragments.len());
-        fragments.retain(is_fragment_valid);
+        fragments.retain(|fragment| {
+            let res = is_fragment_valid(fragment);
+            let id = fragment.id();
+            if res {
+                tracing::debug!(id=?id, "received valid fragment");
+            } else {
+                tracing::debug!(id=?id, "invalid fragment not included into the pool");
```
Comment (mzabaluev) [inline jormungandr/src/fragment/pool.rs:90]: Should we have a common span with an identifier for these two events?

**Comment by eugene-babichenko** [comment]
I am going to redo this in #3301

File: jormungandr/src/fragment/pool.rs (hunk 1 above)
Comment (eugene-babichenko) [reply jormungandr/src/fragment/pool.rs:90]: Yes, good catch.

**Comment by eugene-babichenko** [comment]
Superseded by #3301

**Pull request closed without merging**